### ISSUE TITLE
django-memoize vs. memoize

### ISSUE THREAD

**Issue by anx-ckreuzberger** [tvavrys/django-memoize#23]
There is a pypi package called "memoize" (which exists since at least 2013), while there also is "django-memoize" (this package). However, the module in this package is called "memoize", which is conflicting with the existing pypi package. 

One could accidentally do a "pip install memoize", which would result in the following error message when you run your code:
```bash
from memoize import memoize, delete_memoized, delete_memoized_verhash
ImportError: cannot import name 'memoize'
```

Though the worst part of this is when you consider recent squatting events on pypi, where people register "fake" packages with malicious code (see https://www.pytosquatting.org/ for more details, in case you missed it).

There is no imminent threat for django-memoize, but I would like to begin a discussion about renaming the module in this package to "django_memoize", so that it matches the package name.

Any thoughts on this from others?

**Comment by unformatt** [comment]
Agreed. I see `memoize` in my code. I see `memoize` in my site-packages. So I search internet for `python memoize github`. I find https://github.com/rlgomes/memoize. Very confusing.

**Comment by jimmy927** [comment]
> One could accidentally do a "pip install memoize", which would result in the following error message when you run your code:

I just did that and got very confused until i found this post :-)

**Comment by anx-ckreuzberger** [comment]
That means this post can now be found via search engines :D

**Comment by PetrDlouhy** [comment]
This means, that `django-memoize` and `memoize` can't be used in one project even as dependency of other module. The `django-admin-charts` broke other application when I added `django-memoize` as dependency:
https://github.com/PetrDlouhy/django-admin-charts/commit/758b5809483872eb81e83491ae0617019c87fd42#r87232967
